.github/conda/meta.yaml

```diff
@@ -1,56 +0,0 @@
-{% set name = "transformers" %}
```

Comment:
this file is 4yr old and has very old requirements -- I'm assuming this was used in an older CI :D